readme.md

```diff
@@ -139,9 +139,9 @@ Internally, the standard methods (`GET`, `POST`, `PUT`, `PATCH`, `HEAD` and `DEL
 
 ##### json
 
-Type: `object`
+Type: `any`
```

Comment:
Maybe instead of an actual type (which would be very verbose), we should just say "Any valid JSON value", linking to a reference on MDN or elsewhere that specifies what that is.

Reply:
this is where I can't contribute, as I have no ts experience. I'm happy to go with whatever y'all decide...

Reply:
This is just the documentation for the API, my comment isn't TypeScript related.